metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/feed/JcrFeedProvider.java

```diff
@@ -708,6 +728,66 @@ public void populateInverseFeedDependencies() {
                 .forEach(depFeed -> depFeed.addUsedByFeed(feed1));
         });
     }
+    
+    @Override
+    protected void appendJoins(StringBuilder bldr, String filter) {
+        if (! Strings.isNullOrEmpty(filter)) {
+            bldr.append("JOIN [tba:feedSummary] AS fs ON ISCHILDNODE(fs, e) ");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "JOIN [tba:feedSummary] AS fs ON ISCHILDNODE(fs, e) " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)